android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -827,6 +1057,7 @@ public void createDataChannel(final int peerConnectionId, String label, Readable
             if (-1 != dataChannelId) {
                 dataChannel.registerObserver(new DataChannelObserver(dataChannelId, dataChannel));
                 mDataChannels.put(dataChannelId, dataChannel);
+                registerDataChannelObserver(dataChannelId, dataChannel);
             }
```

Comment:
Nope, it looks different here https://github.com/jitsi/react-native-webrtc/blob/master/android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java
Something went wrong while merging ?